### DISCUSSION TITLE
Limiting Faults

### DISCUSSION THREAD

**Discussion by JoshTheMenace** [gempy-project/gempy#615] [Q&A]
I have gone through the tutorials about faults, and I don't see a way to limit the bounds of a fault. The faults always seem to span across the entire z axis, from top to bottom. 
How can you limit the fault to only spanning across certain surfaces?

**Comment by javoha** [comment]
Hello @TheArcher1958,

faults in GemPy are series (as stratigraphic series) that have ordered time relations. By default faults, only affect older series, younger series (wether they are younger layers or other faults) will not be displaced. This example notebook at least shows this realtion for two faults:
https://docs.gempy.org/tutorials/ch1_fundamentals/ch1_5_fault_relations.html#sphx-glr-tutorials-ch1-fundamentals-ch1-5-fault-relations-py

I will move this topic to discussions, as I dont think it's a bug. Feel free to ask more questions or post your specific problem if you need help.
